qiskit/algorithms/gradients/__init__.py

```diff
@@ -48,8 +48,9 @@
 .. autosummary::
```

Comment:
I can't comment there, but should we change this section name on L46 from ``QFI`` to ``Quantum Geometric Tensor``?